### DISCUSSION TITLE
Timeline for releasing Med-Align database

### DISCUSSION THREAD

**Discussion by hanyin88** [som-shahlab/medalign#3] [General]
Hello, 

Thanks for the exceptional work in creating MedAlign. I believe this work will significantly promote the field. 

I am hoping to ask about a rough timeline on the release of MedAlign database. No rush at all, and I am just hoping to see whether it would make sense to plan on testing our research project on MedAlign.

Many thanks.
Hanyin

**Comment by m-cluitmans** [comment]
I would also be interested in the release time line! These data carry much value.

Thanks,
Matthijs

**Comment by burglarhobbit** [comment]
26 citations already without an actual dataset release. That too with all the (paid?) advertisements through ML influencers on Twitter and LinkedIn. This field is cooked. There is no incentive anymore. It's not happening,
